dev-docs/bidders/slimcut.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: bidder
+title: Slimcut
+description: SlimCut Bidder Adaptor
+hide: true
+biddercode: slimcut
+media_types: video, banner
+gdpr_supported: true
+---
+
+### Overview
+This documentation covers the parameters for the **SlimCut** bidder adapter. And is intended to be referenced by publishers using prebid 1.x.
```

Comment:
How about Prebid 2.x?  Maybe you mean "Prebid.js 1.0 and later"?

Reply:
Yes my bad, I forgot to include Prebid 2